src/commands/generate/app.ts

```diff
@@ -122,17 +122,44 @@ export default class GenerateApp extends BaseCommand<typeof GenerateApp> {
           },
           name: `${config.name} [id: ${config.configurationId}]`,
         }))
+        choices.push({
+          value: {
+            id: 'new-config',
+            auth: undefined,
+          },
+          name: 'Create new login config',
+        })
         const selectedConfig = await select({
           message: 'Select a login configuration to use in your reference application',
           choices,
         })
-        const clientSecret = validateInputLength(
-          await password({
-            message: "What is the login configuration's client secret?",
-            mask: true,
-          }),
-          INPUT_LIMIT,
-        )
+        let newConfigClientSecret = undefined
+        // Create a new login config
+        if (selectedConfig.id === 'new-config') {
+          const newConfigName = validateInputLength(
+            await input({ message: `Enter a name for the login config` }),
+            INPUT_LIMIT,
+          )
+          const redirectUri = getRedirectUri(GenerateApp.apps, appName)
+          const createLoginConfigInput = {
+            name: newConfigName,
+            redirectUris: [redirectUri],
+          }
+          const createConfigOutput = await vpAdapterService.createLoginConfig(createLoginConfigInput)
+          selectedConfig.id = createConfigOutput.configurationId
+          selectedConfig.auth = createConfigOutput.auth
+          newConfigClientSecret = createConfigOutput.auth.clientSecret
```

Comment:
If we created the config, let's print out the client secret so that the user can use it later on and remembers to store it.

Reply:
sure, i'll add this.
Three options, which one would be better?
1. only the client secret
2. output from create login config
3. client id and secret

Reply:
Complete output is a bit too big to print out during the middle of the flow. I'll say option 3 with the same warning to copy it and store it safely.

Reply:
added the above with issuer as well
![Uploading Screenshot 2024-01-04 at 7.50.06 PM.png…]()

Reply:
added the above with issuer as well
<img width="771" alt="Screenshot 2024-01-04 at 7 50 06 PM" src="https://github.com/affinidi/affinidi-cli/assets/146182581/d6a9d692-ada2-4988-9b66-a517fb702de6">